src/Middleware/HttpOverrides/src/ForwardedHeaders.cs

```diff
@@ -30,7 +30,11 @@ public enum ForwardedHeaders
     /// </summary>
     XForwardedPrefix = 1 << 3,
     /// <summary>
+    /// Process X-Forwarded-Port, which identifies the port used by the client.
+    /// </summary>
+    XForwardedPort = 1 << 3,
```

Comment:
We want these flags to be unique (right now it's the same value as the prior one). More info about Flags here: https://learn.microsoft.com/en-us/dotnet/api/system.flagsattribute?view=net-7.0

Reply:
```suggestion
    XForwardedPort = 1 << 4,
```
Oops copy pasta